File: python-client-sdk/src/sdk/models/webhooks/newpet.py
import dataclasses
from ..shared import newpet as shared_newpet
from typing import Optional


@dataclasses.dataclass
class NewPetResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    

@dataclasses.dataclass
class NewPetRequest:
    request: Optional[shared_newpet.NewPet] = dataclasses.field(default=None, metadata={'request': { 'media_type': 'application/json' }})
    

File: typescript-client-sdk/src/sdk/models/webhooks/newpet.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import * as shared from "../shared";
import { Type } from "class-transformer";


export class NewPetResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}

export class NewPetRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "request, media_type=application/json" })
  request?: shared.NewPet;
}